package.json

```diff
@@ -67,7 +67,7 @@
   },
   "dependencies": {
     "@konveyor/lib-ui": "^1.4.0",
-    "@patternfly/react-core": "^4.40.4",
+    "@patternfly/react-core": "4.47.1",
```

Comment:
Please split the Patternfly update into a separate PR. Thanks

Reply:
https://github.com/konveyor/virt-ui/pull/26